oop1-construct.js

```diff
@@ -81,7 +168,7 @@ circle.calcArea();
 Bonus Exercises:
 
 //4. Create three more instances of the Shape class for a pentagon, hexagon, and an octagon.  Add in  comments explaining the formulas for calculating the perimeter and the areas of these 3 shapes.  Also, as noted above, if you use the formula to calculate the height of the triangle instead of hard coding it, you will also obtain the bonus for this exercise.  The dimensions for the triangle are already provided for you in the triangle object literal.
-
+//see triangle formula
```

Comment:
I don't see anything for pentagons, hexagons, or octagons in the triangle section, nor comments explaining the formulas...am I missing it?